tests/st/utils/network.py

```diff
@@ -51,6 +51,18 @@ def delete(self):
         """
         self.init_host.execute("docker network rm " + self.name)
 
+    def disconnect(self, host, container):
+        """
+        Disconnect container from network
+        :return: Nothing
+        """
+        args = [
```

Comment:
I don't like the whole args and join thing. Just write it out as a single string
host.execute("docker network disconnect %s %s" % (self.name, container))

You also don't need the rstrip()